pkgs/top-level/python-packages.nix

```diff
@@ -1159,6 +1159,35 @@ in modules // {
     };
   };
 
+  aws_shell = buildPythonPackage rec {
+    name = "aws-shell-${version}";
+    version = "0.1.0";
+    src = pkgs.fetchFromGitHub {
```

Comment:
Please use releases in in archives when available. In this case, 
https://pypi.python.org/pypi/aws-shell/0.1.0